src/database/models/user.js

```diff
@@ -58,5 +58,13 @@ module.exports = (sequelize, DataTypes) => {
       foreignKey: 'followeeId'
     });
   };
+
+  User.associate = models => {
+    // associations can be defined here
```

Comment:
Also this.

Reply:
Thanks for this review.
Can you point explicitly to what you are referring to as this

Reply:
I think she meant the comment. Let's ensure all unused variables and stubs are removed.

Reply:
Noted